resources/urn/db/snemo_simulation_setup_infos.defs

```diff
@@ -72,11 +80,11 @@ description : string = "SuperNEMO demonstrator simulation setup (version 2.1)"
 
 topics : string[5] = "expsetup" "vertexes" "decays" "variants" "services"
 
-topic.expsetup.component   : string = "urn:snemo:demonstrator:setup:1.0"
-topic.vertexes.component   : string = "urn:snemo:demonstrator:simulation:vertexes:4.1"
-topic.decays.component     : string = "urn:snemo:demonstrator:simulation:decays:1.2"
-topic.variants.component   : string = "urn:snemo:demonstrator:simulation:2.1:variants"
-topic.services.component   : string = "urn:snemo:demonstrator:simulation:2.1:services"
+topic.expsetup.component : string = "urn:snemo:demonstrator:setup:1.0"
+topic.vertexes.component : string = "urn:snemo:demonstrator:simulation:vertexes:4.1"
+topic.decays.component   : string = "urn:snemo:demonstrator:simulation:decays:1.2"
+topic.variants.component : string = "urn:snemo:demonstrator:simulation:2.1:variants"
+topic.services.component : string = "urn:snemo:demonstrator:simulation:2.1:services"
 
 
 # end
```

Comment:
Where is `decays:1.3` in the final setup?

Reply:
Tag "urn:snemo:demonstrator:simulation:2.1" is built on "decays:1.2"  and will be forever.
This should never be changed. The dependancy relationship between tags, once initialized, is frozen.
If you need a new simu setup that used "decays:1.3", then we will create a new tag for that
with new links to tagged modules. The former one will be usabled as before. And in parallel, 
the new one is published for users whio requested it.
We have two choices here:
1) A new tag "urn:snemo:demonstrator:simulation:2.2" 
can be created: it will use the same modules than 2.1 but will link to the new "decays:1.3" in place of "decays:1.2"
It is a minor update because the decay generator is independant of the rest of the MC setup (geometry/vertex/g4).

This option is immediately feasable. It is just an integration effort in this PR or  the next one
after validation of this work.

2) A new tag  "urn:snemo:demonstrator:simulation:3.0" 
will be created in the future: with brand new components:
- geometry 5.0 (not ready yet, in progress): new realistic source foils, BB materials...
- vertex 5.0 (not ready yet, in progress, needs geo 5.0) : new vtx generators
- decay 1.3 (ready, this work)

We believe such work must be split in several PRs because the components
to be addressed are complex and there are strong deps between them. Unit tests
will be easier if we work step by step.
From a new more realistic geometry available for 3D rendering but not MC yet,
users can have new ideas and requests for vtx gnrtors.
Proposal is:
- 1 PR for a preliminary geom 5.0 
- 1 PR for a preliminary vtx 5.0
- N PRs for finalization of geom/vtx 5.0/5.0
- 1 PR for a MC setup 3.0 (integration of  former PRs)
- new Falaise tag may occur here

Reply:
As per my comment in #82 , this is arguing for splitting the management of resources into separate repos! Not for this PR, but again discussion in #33 is required!